tests/auth0/handlers/rules.tests.js

```diff
@@ -74,6 +74,44 @@ describe('#rules handler', () => {
       }
     });
 
+    it('should not collision order when rules are reordered', async () => {
```

Comment:
This wording isn't quite right. It should be either `should not collide` or `should not have a collision`